src/plugins/mer/mertoolchain.cpp

```diff
@@ -157,18 +157,17 @@ QList<Task> MerToolChain::validateKit(const Kit *kit) const
     return result;
 }
 
-ProjectExplorer::HeaderPaths MerToolChain::builtInHeaderPaths(const QStringList &cxxflags, const FileName &sysRoot) const
-{
-    Q_UNUSED(cxxflags)
-    if (m_headerPathsOnHost.isEmpty()) {
-        m_headerPathsOnHost.append(HeaderPath(sysRoot.toString() + QLatin1String("/usr/local/include"),
-                                             HeaderPathType::System));
-        m_headerPathsOnHost.append(HeaderPath(sysRoot.toString() + QLatin1String("/usr/include"),
-                                             HeaderPathType::System));
-    }
-    return m_headerPathsOnHost;
-}
-
+//ProjectExplorer::HeaderPaths MerToolChain::builtInHeaderPaths(const QStringList &cxxflags, const FileName &sysRoot) const
```

Comment:
Please remove commented code before removing *WIP* :)

Reply:
Yup, I just need to recollect why I commented it out :)